lib/rubocop/lockfile.rb

```diff
@@ -5,6 +5,13 @@ module RuboCop
   # Does not actually resolve gems, just parses the lockfile.
   # @api private
   class Lockfile
+    # @param [String, Pathname, nil] lockfile_path
+    def initialize(lockfile_path: nil)
+      lockfile_path ||= defined?(Bundler) ? Bundler.default_lockfile : nil
```

Comment:
I parameterized this class to let you specify your own `lockfile_path`, but kept the default behaviour the same as before.

This let's use it with the result from `RuboCop::Config#bundler_lock_file_path`.